cost/google/unused_cloudsql_instances/README.md

```diff
@@ -0,0 +1,33 @@
+# Google Unused CloudSQL Instances
+
+## What it does
+
+This Policy Template checks for unused CloudSQL instance in Google Compute Engine and then terminates them upon approval.
+
+## Functional Details
+
+- This policy identifies all CloudSQL reporting performance metrics to Google StackDriver and delivers a report, for instances whose connections are below the thresholds set in the **Number of connections to consider unused** parameters. These thresholds are what you would consider to be and idle instance.
```

Comment:
some changes.  
This policy uses the GCP API to identify unused CloudSQL instances using  performance metrics from Google StackDriver and delivers a report for instances whose connections are below the thresholds set in the **Number of connections to consider unused** parameters. These thresholds are what you would consider to be an used instance."